dts/arm/nordic/nrf54l09_enga_cpuapp.dtsi

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54l09.dtsi>
+#include "nrf54l_05_10_15_cpuapp.dtsi"
```

Comment:
Either the file should be renamed or separate file for 09 app core should be added.

Reply:
I have inlined the contents from `#include "nrf54l_05_10_15_cpuapp.dtsi"`, similar to how it was done in nrf54l20. Let me know if you want this to be done some other way.